examples/platforms/simulation/system.c

```diff
@@ -63,56 +64,101 @@ static void handleSignal(int aSignal)
     gTerminate = true;
 }
 
+/**
+ * This enumeration defines the argument return values.
+ *
+ */
+enum
+{
+    OT_SIM_OPT_HELP        = 'h',
+    OT_SIM_OPT_SLEEP_TO_TX = 's',
+    OT_SIM_OPT_TIME_SPEED  = 't',
```

Comment:
On POSIX platform, this is `s`. Is it better to use the same?

Reply:
Updated.